### ISSUE TITLE
Reformat Nutrition tab

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#2449]
![54566761-1709f880-49d2-11e9-9fca-c614857bb013](https://user-images.githubusercontent.com/1689815/54604537-44dd5480-4a47-11e9-9bc8-6ad6488113ac.png)

**Comment by VaiTon** [comment]
I don't think that putting the serving size in that table is the right thing to do.
That's because if we don't have nutrition values the table is not displayed, and the serving size is not displayed too even if we do have it.
